static/app/components/codeSnippet.tsx

```diff
@@ -116,7 +123,7 @@ export function CodeSnippet({
       onLoad: () =>
         Prism.highlightElement(element, false, () => onAfterHighlight?.(element)),
     });
-  }, [children, language, onAfterHighlight]);
+  }, [children, language, onAfterHighlight, lineHighlightLoaded]);
```

Comment:
Should this effect bail out `if (!lineHighlightLoaded)` or do we mean to treat this as a progressive enhancement?